rudra/data_store/bigquery/maven_bigquery.py

```diff
@@ -41,7 +41,9 @@ def __init__(self, big_query_instance=None, s3_client=None,
         self.big_query_instance = big_query_instance or MavenBigQuery()
         self.big_query_content = list()
         self.counter = Counter()
-        self.bucket_name = 'developer-analytics-audit-report'
+        self.bucket_name = self.s3_client.bucket_name\
```

Comment:
self.bucket_name = self.s3_client.bucket_name or 'developer-analytics-audit-report'

Reply:
oops that is a better way indeed, thanks will do!